utils/db.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable no-console */
+'use strict';
+
+const { Sequelize } = require('sequelize');
+
+const sequelize = new Sequelize('postgres', 'postgres', '4426996VFRC', {
```

Comment:
better to create constants
```suggestion
const DB_NAME = 'postgres';
//...

const sequelize = new Sequelize(DB_NAME, 'postgres', '4426996VFRC', {
```